src/integration-test/java/com/commercetools/sync/inventories/utils/InventoryUpdateActionUtilsItTest.java

```diff
@@ -0,0 +1,210 @@
+package com.commercetools.sync.inventories.utils;
```

Comment:
the utils package in the integration tests is meant for utilities for the integration tests..